2.5/test/run

```diff
@@ -181,6 +181,27 @@ for server in ${WEB_SERVERS[@]}; do
 
   info "All tests for the ${server}-test-app finished successfully."
   cleanup ${server}
+
 done
 
+# test from dockerfile examples
+pushd ${test_dir}
+  info "Check building using a Dockerfile"
+  ct_test_app_dockerfile ../../examples/from-dockerfile/2.5/Dockerfile 'https://github.com/sclorg/rails-ex.git' 'Welcome to your Rails application on OpenShift' app-src
+  t1=$?
+  if [ "$t1" == "0" ]; then
+		echo "test 1 from_dockerfile passed";
```

Comment:
Please also note we're not ouputting anything for passing tests.

Reply:
Ok I will remove it, but in such a huge test log full of various outputs, isn't it useful for debugging?

Reply:
if yes, we should have a generic method, not a custom output for every test.